core/org.wso2.carbon.ui/src/main/java/org/wso2/carbon/ui/CarbonUILoginUtil.java

```diff
@@ -269,10 +273,12 @@ protected static boolean handleLogout(CarbonUIAuthenticator authenticator,
 				session.removeAttribute("logged-user");
 				session.getServletContext().removeAttribute("logged-user");
 				try {
-					session.invalidate();
+					invalidateSession(session);
 				} catch (Exception ignored) {
-				}
-				log.debug("Frontend session invalidated");
+                    if (log.isDebugEnabled()) {
```

Comment:
It seems the above block needs some formatting. Lines are not indented properly.